statics/src/test/java/integration/BrowserOptionsInterceptorSampleUsageTest.java

```diff
@@ -0,0 +1,17 @@
+package integration;
+
+import com.codeborne.selenide.Configuration;
+import org.junit.jupiter.api.Test;
+
+class BrowserOptionsInterceptorSampleUsageTest extends IntegrationTest {
+  @Test
+  void canModifyOptionsByConfig() {
+    Configuration.browserOptionsInterceptors.chromeOptionsInterceptor = options -> {
```

Comment:
Suggestion for naming: 
`Configuration.extraBrowserOptions.chrome` or `Configuration.additionalBrowserOptions.chrome`

Reply:
Seems it not just add extra options but also can override it...